kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java

```diff
@@ -1034,10 +1034,21 @@ private void encryptPlainSnapshots() throws KuraException, IOException {
         }
     }
 
-    private synchronized long saveSnapshot(List<ComponentConfiguration> configs) throws KuraException {
+    private synchronized long saveSnapshot(List<ComponentConfiguration> configs, boolean includeDefinition)
```

Comment:
Do we need to keep the "includeDefinition" parameter? All the usages we have right now set the value to false.
is there any other usage that I've missed?

Reply:
I'd keep it, at the moment no method set the flag to true but maybe in the future some methods will do it, also a configuration normally includes the definition so am invoker could expect to have the configuration definition serialized.

Reply:
Correct.
But that is a private method. So, users will not be able to leverage it in any way, right?

Reply:
that's right, it's more a concern about someone who could use the method in the future, but however we can remove the flag

Reply:
Thanks.